ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/offers/PartyStatementMapperTest.java

```diff
@@ -83,47 +88,68 @@ public void mapCCDPartyStatementToDomain() {
 
         //then
         ListIterator<PartyStatement> partyStatementIterator = partyStatements.listIterator();
-        ListIterator<CCDPartyStatement> ccdPartyStatementIterator = ccdPartyStatements.listIterator();
+
+        ListIterator<CCDCollectionElement<CCDPartyStatement>> ccdPartyStatementIterator
+            = ccdPartyStatements.listIterator();
 
         while (partyStatementIterator.hasNext()) {
-            assertThat(ccdPartyStatementIterator.next()).isEqualTo(partyStatementIterator.next());
+            assertThat(ccdPartyStatementIterator.next().getValue()).isEqualTo(partyStatementIterator.next());
         }
     }
 
     @Test
     public void mapEmptyCCDPartyStatementToDomain() {
         //given
-        final CCDPartyStatement ccdPartyStatement = CCDPartyStatement.builder().build();
+        CCDPartyStatement ccdPartyStatement = CCDPartyStatement.builder().build();
+        String collectionId = UUID.randomUUID().toString();
+        CCDCollectionElement<CCDPartyStatement> collectionElement = CCDCollectionElement.<CCDPartyStatement>builder()
+            .id(collectionId)
+            .value(ccdPartyStatement)
+            .build();
 
         //when
-        PartyStatement partyStatement = partyStatementMapper.from(ccdPartyStatement);
+        PartyStatement partyStatement = partyStatementMapper.from(collectionElement);
```

Comment:
We cannot use the PartyStatementAssert for both to and from. It wont test properly. This needs change.

Reply:
point is correct, can be a tech debt..

Reply:
Please create a Tech Debt and comment here, just to agree. Thanks